src/containerTest/kotlin/com/quipalup/katydid/UpdateLogEntryByIdCT.kt

```diff
@@ -0,0 +1,92 @@
+package com.quipalup.katydid
+
+import com.quipalup.katydid.common.id.Id
+import com.quipalup.katydid.logentry.domain.LogEntry
+import com.quipalup.katydid.logentry.domain.LogEntryRepository
+import io.restassured.RestAssured
+import io.restassured.module.kotlin.extensions.Extract
+import io.restassured.module.kotlin.extensions.Given
+import io.restassured.module.kotlin.extensions.Then
+import io.restassured.module.kotlin.extensions.When
+import java.util.UUID
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.web.server.LocalServerPort
+
+@SpringBootTest(
+    classes = [Katydid::class],
+    webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT
+)
+class UpdateLogEntryByIdCT {
+
+    @Autowired
+    private lateinit var repository: LogEntryRepository
+
+    @LocalServerPort
+    private val port: Int = 0
+
+    @BeforeEach
+    fun setUp() {
+        RestAssured.port = port
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails()
+    }
+
+    private val uuid: String = "e903d71b-234b-4129-996e-a6b411f2862d"
+
+    @Test
+    fun `update log entry by id`() {
+
+        repository.create(LogEntry(
+            id = Id(UUID.fromString(uuid)),
+            time = 123345534,
+            description = "Yogurt with strawberries",
+            amount = 100,
+            unit = "percentage"
+        ))
+        Given {
+            contentType("application/vnd.api+json")
+            body(requestBody)
+            pathParam("id", uuid)
+        } When {
+            patch("/log-entries/{id}")
+        } Then {
+//            contentType("application/vnd.api+json")
+            statusCode(200)
+        } Extract {
+            `assert that response body is equal to`(body().asString(), expectedResponseBody)
+        }
+    }
+
+    private val requestBody: String = """
+            {
+              "data": 
+                  {
+                    "type": "meal-log-entry",
```

Comment:
```suggestion
                    "id": "$uuid"
                    "type": "meal-log-entry",
```
https://jsonapi.org/format/#crud-updating

_The PATCH request MUST include a single resource object as primary data. The resource object MUST contain type and id members._

```
PATCH /articles/1 HTTP/1.1
Content-Type: application/vnd.api+json
Accept: application/vnd.api+json

{
  "data": {
    "type": "articles",
    "id": "1",
    "attributes": {
      "title": "To TDD or Not"
    }
  }
}
```